lepidopter-vmdebootstrap_build.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -exa
+GIT_BUILD="$(git describe)"
```

Comment:
Nice! You can also consider `git describe --tags` that yields `${tag}` on release and `${tag}-${number-of-commit-since-tag}-${begin-of-hash-of-latest-commit}` otherwise